packages/example/src/Lottie/Cybertruck/index.tsx

```diff
@@ -0,0 +1,44 @@
+import {Lottie} from '@remotion/lottie';
+import {useEffect, useState} from 'react';
+import {
+	AbsoluteFill,
+	continueRender,
+	delayRender,
+	useVideoConfig,
+} from 'remotion';
+
+const LottieCybertruck = () => {
+	const {height, width} = useVideoConfig();
+	const [animationData, setAnimationData] = useState(null);
+	const [handle] = useState(delayRender);
+
+	useEffect(() => {
+		// Credits: https://lottiefiles.com/11643-tesla-cybertruck
+		fetch('https://assets2.lottiefiles.com/packages/lf20_RqpTFh.json')
+			.then((res) => res.json())
+			.then(setAnimationData);
+	}, []);
+
+	useEffect(() => {
```

Comment:
Should we align the examples and the documentation regarding using two effects or just one? I personally prefer to break them but if there's no risk in calling `continueRender` after setting the state instead of after the component rerenders, I'm fine.

Reply:
It should always work if you call it directly because Remotion also waits for the JavaScript event loop to complete. I updated it!